mathesar_ui/src/systems/table-view/table-inspector/table/TableActions.svelte

```diff
@@ -40,42 +30,40 @@
       },
     });
   }
+
+  $: explorationPageUrl =
+    $currentDatabase && $currentSchemaId
+      ? createDataExplorerUrlToExploreATable(
+          $currentDatabase?.name,
+          $currentSchemaId,
+          $tabularData.id,
+        )
+      : '';
 </script>
 
 <div class="actions-container">
-  <Button appearance="plain" on:click={() => linkTableModal.open()}>
-    <Icon {...iconTableLink} />
-    <span>Link Table</span>
-  </Button>
-  <LinkTableModal
-    controller={linkTableModal}
-    on:goToConstraints={() => tableConstraintsModal.open()}
-  />
-
   {#if $currentDatabase && $currentSchemaId}
-    <Button appearance="plain">
-      <Icon {...iconExploration} />
-      <a
-        class="btn-link"
-        href={getDataExplorerPageUrl($currentDatabase.name, $currentSchemaId)}
-        >Explore Data</a
+    <ActionItem href={explorationPageUrl}>
+      <Icon {...iconExploration} /> <span>Explore Data</span>
+      <Help
+        >Open this table in Data Explorer to query and analyze your data.</Help
       >
```

Comment:
This is very minor, but here's a case where I don't like Prettier's auto-formatting choices. Prettier is trying to ensure that the code does not have any white space between the opening tag and the first word, or between the last word and the closing tag. That sort of thing _can_ matter, but only in very rare cases. So when I end up with code that Prettier breaks up in this way, I change it to the following, which I find more readable:

```html
<Help>
  Open this table in Data Explorer to query and analyze your data.
</Help>
```

Reply:
Looks good. Thanks.